README.md

```diff
@@ -29,40 +31,58 @@ export default {
   name: 'FormExample',
   components: {
     Form
+  },
+  data() {
+    return {
+      form: {
+        email: '',
+        password: ''
+      }
+    }
+  },
+  methods: {
+    handleSuccessSubmit(values, actions) {
+
+      // Here handle form submit and then call setSubmitting()
+
+      actions.setSubmitting(false)
```

Comment:
Shouldn't be `actions.setSubmitting(true)`?

Reply:
No. `Form` sets `props.isSubmitting` after it successfully validates form and before it calls your success handler - `handleSuccessSubmit`. In future I expect it to handle `isSubmitting(false)` based on return value from user handler function which when returning `Promise` could be handled automatically for user.